packages/terra-dropdown-button/tests/wdio/dropdown-button-spec.js

```diff
@@ -109,19 +109,19 @@ Terra.describeViewports('Dropdown Button', ['medium'], () => {
         Terra.validates.element('click');
       });
 
-      it('keyboard navigates down past the last option', () => {
-        browser.keys(['ArrowDown', 'ArrowDown', 'ArrowDown', 'ArrowDown', 'Space']);
-        Terra.validates.element('no down out of bounds');
+      it('Navigates from last item to first item', () => {
+        browser.keys(['ArrowDown', 'ArrowDown', 'ArrowDown', 'ArrowDown', 'Enter']);
+        Terra.validates.element('first item focused');
```

Comment:
Can we also test for focus using `expect`? It will be something like this (replace " first list item" with the appropriate selector):
```suggestion
        expect($(['first list item']).isFocused()).toBeTruthy();
        Terra.validates.element('first item focused');
```

Reply:
Added test case to focus first list item.